UnitystationLauncher/Models/Config.cs

```diff
@@ -35,6 +35,9 @@ static class Config
         public static FileSystemWatcher FileWatcher { get; }
         public static IObservable<Unit> InstallationChanges { get; }
 
+        public static string SiteUrl = "https://unitystation.org/";
+        public static string SupportUrl = "https://www.patreon.com/unitystation";
+        public static string ReportUrl = "https://github.com/unitystation/unitystation/issues";
```

Comment:
Is there a reason these aren't `const`? Also, public fields shouldn't be used, make them properties.
```suggestion
        public const string SiteUrl = "https://unitystation.org/";
        public const string SupportUrl = "https://www.patreon.com/unitystation";
        public const string ReportUrl = "https://github.com/unitystation/unitystation/issues";
```